nym-vpn-app/src-tauri/src/log.rs

```diff
@@ -58,10 +58,12 @@ pub async fn setup_tracing(log_file: bool) -> Result<Option<WorkerGuard>> {
         info!("logging to file: {}", log_file.display());
         Ok(Some(guard))
     } else {
-        tracing_subscriber::fmt()
-            .with_env_filter(filter)
-            .compact()
-            .init();
+        let mut sub = tracing_subscriber::fmt().with_env_filter(filter).compact();
+        if cfg!(windows) && win_console {
```

Comment:
See previous comment, no need for cfg(windows) here if you make the flag itself cfg(windows)